graphql/README.md

```diff
@@ -0,0 +1,62 @@
+# GraphQL at thoughtbot
```

Comment:
I would remove the "at thoughtbot" instances. The guide is under the thoughtbot org, and I don't believe these approaches are specific to thoughtbot's needs.

Reply:
👍